tests/cli/test_datasets.py

```diff
@@ -417,7 +417,7 @@ def test_relative_git_import_to_dataset(tmpdir, runner, project, client):
         ],
         catch_exceptions=True,
     )
-    assert 2 == result.exit_code
+    assert 1 == result.exit_code
```

Comment:
Exit code 2 represents user miss-use which is the guidance which we tried to follow. Why changing it here now? Do we want to migrate all user miss-use errors to general errors?

Reply:
Didn't know about this. Good point! I reverted back all these to return 2.